quipucords/scanner/network/connect_callback.py

```diff
@@ -65,6 +68,7 @@ def v2_runner_on_unreachable(self, result):
 
     def v2_runner_on_failed(self, result, ignore_errors=False):
         """Print a json representation of the result."""
+        input_log.log_ansible_result(result)
```

Comment:
May want to try/catch around this so any IO or other related failure in this flow doesn't kill a scan.

Reply:
Done.